src/session.ts

```diff
@@ -733,7 +838,7 @@ function addTrack(this: WebPhoneSession, remoteAudioEle, localAudioEle): void {
             } else if (!isNoAudio(report)) {
                 this.noAudioReportCount = 0;
             }
-        }, 2000)
+        }, 2000);
```

Comment:
What this magic number means here?

Reply:
This is to detect no audio for 6 seconds -> every 2 sec * 3 counts the code checks for mediastats  before initiating reinvite.

Reply:
You need to name this constant then

Reply:
Added